docs/products/postgresql/howto/pgbouncer-stats.rst

```diff
@@ -0,0 +1,37 @@
+Accessing PGBouncer statistics
+==============================
+
+PGBouncer is used at Aiven for :doc:`../concepts/pg-connection-pooling`.
+
+Get PGBouncer URL
+------------------
+
+PGBouncer URL can be extracted via Aiven Command Line interface with the following command replacing the ``INSTANCE_NAME`` parameter with the name of your instance::
+
+    avn service get INSTANCE_NAME --json | jq -r '.connection_info.pgbouncer'
```

Comment:
Let's mention `jq` with a link to that tool's homepage as well before this command.

Reply:
Added